### PR TITLE
Fix 'user' param when creating a Doctrine connection

### PR DESCRIPTION
**Pull request by liberostelios**
It turns out the the produced Doctrine connetion was returning the database name as 'user' parameter, thus confusing Doctrine's SchemaManager when returning a table's columns.

### CODE REVIEW

**Comment by yajra** [comment]
@liberostelios thanks for the PR. Can you please provide some snippet/steps how to reproduce the bug? I haven't tried using Doctrine yet with the package. Thanks!

**Comment by liberostelios** [comment]
@yajra no problem. I encountered it while using the [xethron/mirgations-generator](https://github.com/Xethron/migrations-generator) package for generating migrations by an existing Oracle schema. It uses Doctrine's DBAL, which exposes the whole issue.

One of the main implications of returning the wrong 'user' parameter is that Doctrine won't find any columns on the tables, as it is trying to find columns from USER_TAB_COLS with the OWNER passed by the 'user' parameter (which should be the username). So, instead of the real schema owner, it looks for columns with the database name as an owner.

TL;DR, you can just install migrations-generator package and try to generate migrations from a current schema. You will get the table names, but no table migration will be created because of this issue as it won't find any columns.

**Comment by yajra** [comment]
Merged and released on v5.2.10. Thanks!
